techniques/applications/ipsPackageRepositories/1.0/metadata.xml

```diff
@@ -0,0 +1,104 @@
+<!--
+Copyright 2020 Normation SAS
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, Version 3.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<TECHNIQUE name="Package sources and settings (IPS)">
+  <DESCRIPTION>This technique configures the IPS package manager for Solaris.</DESCRIPTION>
+
+  <MULTIINSTANCE>true</MULTIINSTANCE>
+  <POLICYGENERATION>separated</POLICYGENERATION>
+  <USEMETHODREPORTING>true</USEMETHODREPORTING>
+
+  <BUNDLES>
+    <NAME>ips_repository_RudderUniqueID</NAME>
+  </BUNDLES>
+
+  <TMLS>
+    <TML name="ipsPackageRepositories"/>
+  </TMLS>
+
+  <TRACKINGVARIABLE>
+    <SAMESIZEAS>IPS_PUBLISHER_NAME</SAMESIZEAS>
+  </TRACKINGVARIABLE>
+
+  <SECTIONS>
+    <SECTION name="Repository">
+      <INPUT>
+        <NAME>IPS_PUBLISHER_NAME</NAME>
+        <DESCRIPTION>Name</DESCRIPTION>
+        <LONGDESCRIPTION>Name of the publisher repository</LONGDESCRIPTION>
+      </INPUT>
+      <INPUT>
+        <NAME>IPS_PUBLISHER_URL</NAME>
+        <DESCRIPTION>URL</DESCRIPTION>
+        <LONGDESCRIPTION>URL of the publisher repository</LONGDESCRIPTION>
+      </INPUT>
+      <SELECT1>
+        <NAME>IPS_PUBLISHER_ENABLED</NAME>
+        <DESCRIPTION>Enabled</DESCRIPTION>
```

Comment:
Maybe "State" more than "Enabled" which a value

Reply:
And option should be "Enabled" and "Disabled" to match generic methods semantics